### ISSUE TITLE
Fix: "Installed version does not match package.json"

### ISSUE THREAD

**Issue by mathiasrw** [dylang/npm-check#30]
TL;DR: **I suggest that `npm-check -u` gives you the option to update the module version required in `package.json` when there is a newer version of a module installed in `node_modules/`**

----

If you somehow get a new version of a module into the 'node_modules/' so the required version in`´package.json` is less than what is installed then running a `npm-check` will give

    Installed version does not match package.json

Great! lets zap that thing - by running `npm-check -u` - but suddenly I get  

    Your modules look amazing. Keep up the great work.

In my logic the `-u` flag will always help you solve whatever issues was found without the `-u` flag.

**Comment by dylang** [comment]
Great bug, this will be addressed in the next release.

**Comment by mathiasrw** [comment]
Wohooo !
